### ISSUE TITLE
Dryer only mode

### ISSUE THREAD

**Issue by tomtom13** [swingerman/ha-dual-smart-thermostat#219]
Hi,
I think (again stupid me for thinking) that it might be useful for people to use this thermostat in DRYer mode only. 
Reason is that is might be dumb to switch on dehumidifier while window OR door is open, while is perfectly OK to switch on heating / cooling with room door open. 
Of-course this functionality might be already there, however I don't see support for "dry" mode only. Also there is a bug in description:
`
The dryer will turn on when the humidity is above the target humidity and the [moist_tolerance](https://github.com/swingerman/ha-dual-smart-thermostat/issues/new#moist_tolerance) is not reached. If the humidity is above the target humidity and the [moist_tolerance](https://github.com/swingerman/ha-dual-smart-thermostat/issues/new#moist_tolerance) is reached the dryer will stop.
`
where second use of "moist_tolerance" is meant to be "dry_tolerance". 
Yeah, I'm one of those dummies that read docs :)

**Comment by swingerman** [comment]
The latest beta supports dry only mode.
Also, you can set the openings_scope to control which HVAC modes are affected by openings.

**Comment by tomtom13** [comment]
Yeah, but what I meant here is a "thermostat" (or more hygrostat) that doesn't control any heaters or aircon - just humidity. Also currently the thermostat doesn't allow to set different scopes of opening for different entities. For example if I care about the door only for cooling, but I care about the window for both cooling and heating, there is no way to set it. I'm not saying that you HAVE to implement that, but since it's absent I think it would be nice to give an option for the thermostat to work in only DRY mode.

**Comment by swingerman** [comment]
This component extends the climate integration in home assistant. It has limitations and cannot be used as a fully featured hygrostat. This also requires that we set a heater entity.
It would be possible to set it up like the AC mode. For example, `dryer_mode`: true. That turns the thermostat into a dryer.

**Comment by tomtom13** [comment]
That would work, honestly I don't care that much whenever I would need to tie dehumidifier to heater entity or what ever. As long as it works - it works ;)

**Comment by tomtom13** [comment]
Slight note: the "ac_mode" is a bit confusing, in current functionality I think it should be called "ac_only_mode". At least for my puny mind ;)

**Comment by tomtom13** [comment]
BTW, maybe also having a "fan only" mode would be great as well.

**Comment by swingerman** [comment]
Related: this use case would be addressed by the auto mode feature planned in #563.
